### PR TITLE
Refactor naming used for jars and publishing

### PR DESCRIPTION
**Pull request by rain-on**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -35,7 +35,7 @@ if (!JavaVersion.current().java11Compatible) {
 def bintrayUser = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER')
 def bintrayKey = project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_KEY')
 def bintrayPackage = bintray.pkg {
-  repo = 'pegasys-repo'
+  repo = 'test-repo'
```
Comment (jframe) [inline build.gradle]: This should stay as 'pegasys-repo'

File: build.gradle [hunk 2]
```diff
@@ -240,6 +240,12 @@ task deploy() {}
 
 subprojects {
 
+  build {
```
Comment (jframe) [inline build.gradle]: Is this this needed? Looks like it was just for testing

File: build.gradle (hunk 1 above)
Comment (rain-on) [reply build.gradle]: agreed

File: build.gradle (hunk 2 above)
Comment (rain-on) [reply build.gradle]: agreed

**Pull request merged**